flight/mcu/lib/teensyShared/Barometer.cpp

```diff
@@ -1,21 +1,33 @@
-#include <ChRt.h>
-#include <MS5611.h>
+#include "ChRt.h"
+#include "MS5611.h"
 
 #include "BarometerSensor.h"
+#include "dataLog.h"
 
-void BarometerSensor::readReadings() { MS->read(12); }
+BarometerSensor barometer{};
 
-float BarometerSensor::getPressure() {
-    pressure = MS->getPressure() * 0.01;
+void BarometerSensor::init() {
```

Comment:
If the header file and class are named "BarometerSensor", let's also name the source file the same thing